vllm/entrypoints/chat_utils.py

```diff
@@ -52,6 +54,23 @@ class ChatCompletionContentPartAudioParam(TypedDict, total=False):
     """The type of the content part."""
 
 
+class VideoURL(TypedDict, total=False):
+    url: Required[str]
+    """
+    Either a URL of the video or a data URL with base64 encoded video data.
+
+    Learn more in the
+    [Vision guide](https://cookbook.openai.com/examples/gpt_with_vision_for_video_understanding).
```

Comment:
I think this is a bit misleading as the OpenAI tutorial passes videos as a list of images, which is different from here where the entire video is being sent as a single item.

Reply:
OK, make sense. This reference is indeed ambiguous, so let's remove it.